packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -329,13 +336,17 @@ export class CdkToolkit {
       latch = 'open';
       debug("'watch' received the 'ready' event. From now on, all file changes will trigger a deployment");
       print("Triggering initial 'cdk deploy'");
+      cloudWatchLogMonitor?.deactivate();
       await deployAndWatch();
+      cloudWatchLogMonitor?.activate();
     }).on('all', async (event: 'add' | 'addDir' | 'change' | 'unlink' | 'unlinkDir', filePath?: string) => {
       if (latch === 'pre-ready') {
         print(`'watch' is observing ${event === 'addDir' ? 'directory' : 'the file'} '%s' for changes`, filePath);
       } else if (latch === 'open') {
         print("Detected change to '%s' (type: %s). Triggering 'cdk deploy'", filePath, event);
+        cloudWatchLogMonitor?.deactivate();
         await deployAndWatch();
+        cloudWatchLogMonitor?.activate();
```

Comment:
I actually liked it more when `deactivate()` and `activate()` where called in `deployAndWatch()`, sorry 😕. I don't like the duplication that we always have to surround the call to `deployAndWatch()` with `activate()` and `deactivate()`.